Detectors/TRD/workflow/src/TRDDigitizerSpec.cxx

```diff
@@ -125,9 +125,7 @@ class TRDDPLDigitizerTask
         std::vector<o2::trd::Digit> digits;                         // digits which get filled
         o2::dataformats::MCTruthContainer<o2::trd::MCLabel> labels; // labels which get filled
         mDigitizer.process(hits, digits, labels);
-        for (auto& digit : digits) {
-          digit.setTimeStamp(irecords[collID].timeNS);
-        }
```

Comment:
Thanks for removing this.

Reply:
@jolopezl you need to approve the changes as well.